doc/content/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Home"
+description: ""
```

Comment:
Please make sure that an explicitly empty description doesn't override the site-wide description

Reply:
What was the conclusion here? I also still see that some files have either a `desc ription: ""` or no description field at all.

Reply:
`description:` only influence the page description and is not used bu Hugo natively. It used by the theme, often when making search.